Mathlib/Algebra/Lie/Basic.lean

```diff
@@ -1001,10 +1001,15 @@ instance : EquivLike (M ≃ₗ⁅R,L⁆ N) M N :=
     coe_injective' := fun f g h₁ h₂ =>
       by cases f; cases g; simp at h₁ h₂; simp [*] }
 
+@[simp] lemma coe_coe (e : M ≃ₗ⁅R,L⁆ N) : ⇑(e : M →ₗ⁅R,L⁆ N) = e := rfl
```

Comment:
Is `norm_cast` allowed here?

Reply:
Nope it complains `norm_cast: badly shaped lemma, rhs can't start with coe`. (I might look into this in the future.)